pkgs/development/compilers/dotnet/print-hashes.sh

```diff
@@ -46,11 +46,117 @@ platform_sources () {
     echo "      $nix_platform = {
         url     = \"$url\";
         sha512  = \"$hash\";
-      }; "
+      };"
     done
     echo "    };"
 }
 
+sdk_packages () {
+    local version=$1
+    local pkgs=( \
```

Comment:
I wonder if theres any way to automatically generate these? If Microsoft adds any nuget packages to the SDK in future releases this would break. 

I also think a comment should be added explaining why we have to include SDK packages here.

Reply:
Unfortunately, I didn't find any canonical source of those internal packages. It would propably require delving deeply into the .NET source code to find a list of all the packages that can be added like that.

This list came from seeing what is commonly added to projects, and searching Nuget.org for "internal projects"

Reply:
I added some comments with explanations, let me know if its enough

Reply:
> Unfortunately, I didn't find any canonical source of those internal packages. It would propably require delving deeply into the .NET source code to find a list of all the packages that can be added like that.

Ah, alright. Considering we couldn't automatically generate it anyway this should be fine than.

> I added some comments with explanations, let me know if its enough

Definitely! I really appreciate the extensive documentation of this :)